github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
well we are filtering for a certain list of PRs, we figure for now printing just the PR number is satisfactory, because we should trust that our logic worked in filtering for the items we requested. Printing all the info that we just filtered (or didnt filter for) would be time consuming and not offer that much help in the way of productivity.